src/N98/Magento/Command/Config/AbstractConfigCommand.php

```diff
@@ -114,10 +115,12 @@ protected function _convertScopeIdParam($scope, $scopeId)
             );
         }
 
-        if (0 >= (int)$scopeId) {
-            throw new InvalidArgumentException(
-                sprintf("Invalid scope parameter, %s is not a positive integer value", var_export($scopeId, true))
-            );
+        if (!$allowZeroScope) {
+            if (0 >= (int)$scopeId) {
```

Comment:
`0 - (bool)$allowZeroScope >= (int)$scopeId`

this will allow the zero scope then. but not -1 (etc). it's also one less if.